api/src/main/java/edu/cornell/mannlib/vitro/webapp/visualization/temporalgraph/TemporalPublicationVisualizationRequestHandler.java

```diff
@@ -41,15 +41,15 @@
 
 public class TemporalPublicationVisualizationRequestHandler implements
 		VisualizationRequestHandler {
-	

```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional update buried in the noise. Please revert these updates.